packages/camera/camera/example/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   path_provider: ^2.0.0
   flutter:
     sdk: flutter
-  video_player: ^2.0.0
+  video_player: ^2.1.4
```

Comment:
Does image_picker/example also need this bump?

Reply:
Good catch, thanks!